src/Akka.Persistence.Sql.Tests.Common/Query/Bugfix344Spec.cs

```diff
@@ -0,0 +1,93 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="Bugfix344Spec.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2023 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using Akka.Configuration;
+using Akka.Persistence.Query;
+using Akka.Persistence.Sql.Config;
+using Akka.Persistence.Sql.Query;
+using Akka.Persistence.Sql.Tests.Common.Containers;
+using Akka.Streams;
+using Akka.Streams.TestKit;
+using FluentAssertions.Extensions;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Akka.Persistence.Sql.Tests.Common.Query
+{
+    public abstract class Bugfix344Spec<T> : Akka.TestKit.Xunit2.TestKit, IAsyncLifetime where T : ITestContainer
```

Comment:
Abstract spec to validate that this issue has been resolved, and that the `ReadJournal` will automatically start up the write journal and wait for it to initialize.